doc/swish/db.tex

```diff
@@ -249,6 +247,12 @@
   SQLITE\_OPEN\_READWRITE \code{SQLITE\_OPEN\_CREATE})}.
 \end{itemize}
 
+\var{db-init} is a procedure expects one argument, a database record
+instance. The return value is ignored. When the \var{mode} is
+\code{create}, and \var{filename} is not a special SQLite filename,
+the default procedure sets \code{journal\_mode} to ``wal'', otherwise,
+no additional initialization occurs.
+
```

Comment:
\var{db-init} is a procedure **that takes** one argument, a database record
instance. The return value is ignored. When the \var{mode} is
\code{create} _{no comma here}_ and \var{filename} is not a special SQLite filename,
the default procedure sets \code{journal\_mode} to ``wal'' _{use semicolon here}_ otherwise,
no additional initialization occurs.

Reply:
I like that version better. Thank you. Updated.